code/datums/status_effects/buffs.dm

```diff
@@ -572,7 +572,7 @@
 		var/mob/living/carbon/human/humi = owner
 		humi.coretemperature = humi.get_body_temp_normal()
 	owner.restoreEars()
-	duration = rand(150, 450) * power
+	duration = rand(150, 450) * 2
```

Comment:
Might wanna avoid hard coding this, so if you're not using power maybe make a new parameter?